core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -556,7 +557,33 @@ public void addSplits(String tableName, SortedSet<Text> splits)
         futures.add(future);
       }
 
-      futures.forEach(CompletableFuture::join);
+      try {
+        futures.forEach(CompletableFuture::join);
+      } catch (CompletionException ee) {
+        Throwable excep = ee.getCause();
```

Comment:
All of the code in this catch block was copied from a previous commit.  This restores some behavior addSplit used to have for translating exceptions in background threads.